MOVIShield.cpp

```diff
@@ -45,11 +45,13 @@
 
 MOVI::MOVI()
 {
+    usehardwareserial=false;
     construct(ARDUINO_RX_PIN, ARDUINO_TX_PIN, false);
 }
 
-MOVI::MOVI(bool debugonoff)
+MOVI::MOVI(uint8_t debugonoff)
```

Comment:
I mention this in the forum post I made about this.  With debugonoff defined as bool, the compiler (Arduino IDE 1.6.7) was accepting &Serial1 to that constructor without a cast even.  This was how it was behaving before I added the extra constructor for HardwareSerial * anyway.  Perhaps we could put debugonoff back to bool and with the explicitly defined HardwareSerial * constructor the compiler would choose the right one?